.eslintrc.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
 		ecmaVersion: 2018,
 	},
 	rules: {
-		'no-warning-comments': 'warn',
+		'no-warning-comments': [2, { 'terms': ['fixme'], 'location': 'start' }],
```

Comment:
```suggestion
		'no-warning-comments': ['warn', { 'terms': ['fixme'], 'location': 'start' }],
```

I don't think we want `fixme` comments to fail the tests